modules/vpn/main.tf

```diff
@@ -59,7 +61,8 @@ module "security_group_vpn" {
   )
 }
 
-data "aws_ami" "ubuntu_20_ami" {
```

Comment:
@ankush-sqops Can we hardcode the AMI id here. 
Reason: When we reapply the module after a month or so, the VPN instance gets terminated and new instance comes up with new base AMI as AWS launches new AMI every now and then.

Reply:
Okay, I'll check and try to hardcode the AMI

Reply:
@anish-squareops  Update the PR, please check the changes.